Packages/vcs/Lib/VTKPlots.py

```diff
@@ -405,6 +408,12 @@ def portrait(self, W, H, x, y, clear):
         else:
             self.renWin.SetSize(W, H)
 
+    def portrait(self, W=-99, H=-99, x=-99, y=-99, clear=0):
+        self.resize_or_rotate_window(W, H, x, y, clear)
+
```

Comment:
@sankhesh this does not work the following snippet demonstrates the problem:
```python
import cdms2
import vcs
import os
f=cdms2.open(os.path.join(vcs.sample_data,"clt.nc"))
s=f("clt")
x=vcs.init()
x.portrait()  # turn page in portrait mode
x.portrait()  # turn page in portrait mode (again but it is already in portrait so it should be doing nothing)
x.plot(s)
x.png("test_portrait_bg")
```

leads to:
![test_portrait_bg](https://cloud.githubusercontent.com/assets/2781425/10065559/baf5e534-623c-11e5-93c7-305fafceb1b1.png)

Essentially you rotate twice